changed version to 0.10.0-alpha.1

Kevsy opened the pull request:
#### What type of PR is this?

Add one of the following kinds:

* subproject management



#### What this PR does / why we need it:

First upload of YAML from previous Edge Cloud sub-project, incorporating the scope of Simple Edge Discovery for Fall 24  Release #3 



#### Which issue(s) this PR fixes:

<!-- Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`. -->

Fixes issue 3 (which will remain pinned/open as it relates to the release scope )

#### Special notes for reviewers:



#### Changelog input

```
 release-note
 Incorporates the changes listed in  #3 
 
 ```

#### Additional documentation 

This section can be blank.



```
docs

```

hdamker commented:
@Kevsy two options here:

1) This is already the PR which creates the alpha release. Then it should contain the CHANGELOG.md and the change to the README. Please add in the CHANGELOG also the link to the YAML within EdgeCloud (the exact commit which you have copied (I suppose that is https://github.com/camaraproject/EdgeCloud/blob/ed32560de80e55943d67766a040962df54d5498b/code/API_definitions/Discovery/simple_edge_discovery.yaml)
In this case you have to set also the URL to `simple-edge-discovery/v0.10alpha.1`

2) Recommend: you use this PR only to transfer the YAML and doing a release PR after some correction. Then the version both in version field and the URL would be "wip".

hdamker commented on code/API_definitions/simple-edge-discovery.yaml:
```diff
@@ -0,0 +1,650 @@
+        default: https://localhost:443
+        description: API root.
+      basePath:
+        default: simple-edge-discovery/v1
```
Need to be aligned with the version number.

hdamker commented on code/API_definitions/simple-edge-discovery.yaml:
```diff
@@ -0,0 +1,650 @@
+     example: 2001:db8:85a3:8d3:1319:8a2e:370:7344
+
+
+###################################
```
These lines are producing a linting error as they are not intended.

Kevsy commented:
Thanks @hdamker , 

> @Kevsy two options here:

> 2. Recommend: you use this PR only to transfer the YAML and doing a release PR after some correction. Then the version both in version field and the URL would be "wip" for now.

Done

> Need to be aligned with the version number.

Done

> These lines are producing a linting error as they are not intended.

Done, removed

Also removed the release tag.

javierlozallu reviewed (approved):
Looks good to me. Perhaps you want to include the SED.test file since both files are in the Edge Cloud repository or do you want to wait for Commonalities 0.4 to upload all the necessary documents?

Kevsy commented:
> Looks good to me. Perhaps you want to include the SED.test file since both files are in the Edge Cloud repository or do you want to wait for Commonalities 0.4 to upload all the necessary documents?

Thanks @javierlozallu  - I'll make a separate PR for that, and probably wait for the Commonalities v0.4 which will  confirm the  instructions in https://github.com/camaraproject/Commonalities/blob/main/documentation/API-Testing-Guidelines.md
